keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,1261 @@
+interface for the CCI Manager which will use data extracted from CCI Store. This will 
+require such data to be provided by CCI Drivers and the CCI store has to be extended to 
+be able to handle it correctly for serving the corresponding topology hints calls.
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+###### Alpha
+
+*	CCI Resource Manager: target code cvg >=80%
+*	CCI Store: target code cvg >=80%
+*	CCI CPU Manager Policy Component: target code cvg >=80%
+*	CCI Drivers Factory API: target code cvg >=80%
+
+###### BETA
+
+*	CPU Manager None and Static Policy Integration with CCI
+* Introduce fail-safety tests
+*	Performance/Scalabilty tests
+
+
+##### Integration tests
+###### Alpha
+* CPU and CCI Manager Integration test: driver-based allocation and best-effort QoS 
+* State Consistency (CPU Manager + CCI) integrateion test
+###### BETA
+* CPU Manager None, Static Policy Integration with CCI
+* CPU, Memory, Topology and CCI Manager Integration test
+*	Further integration tests with Device Manager and DRA
+*	Integration test including static QoS and driver-based allocation
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+- <test>: <link to test coverage>
+-->
+
+##### e2e tests
+###### Alpha
+*	E2E test with mocked CCI driver 
```

Comment:
I'm happy some e2e tests are actually planned for alpha! so please make sure this is consistent with other pieces of the KEP (goals/non-goals come to mind) where it may be given the impression e2e test coverage is not a alpha goal